src/settings/Settings.ts

```diff
@@ -116,6 +116,12 @@ export interface ISetting {
 }
 
 export const SETTINGS: {[setting: string]: ISetting} = {
+    "feature_latex_maths": {
+        isFeature: true,
+        displayName: _td("LaTeX math in messages"),
```

Comment:
Personally I would suggest making it more active with a verb, and avoiding math vs. maths by just removing the word:

```suggestion
        displayName: _td("Render LaTeX in messages"),
```

If you'd like to keep "maths" in there, then please use UK spelling.

Reply:
I decided to go with "Render LaTeX maths in messages". I think saying "maths" here is useful, since all tex formatting is restricted to between the dollar-signs, and general text formatting should (still) be done with markdown.